apps/dh/api-dh/project.json

```diff
@@ -6,64 +6,31 @@
   "prefix": "dh",
   "targets": {
     "build": {
-      "configurations": {
-        "ci": {}
-      },
+      "outputs": ["{projectRoot}/source/DataHub.WebApi/bin/Release/net8.0"],
       "executor": "nx:run-commands",
       "options": {
-        "commands": [
-          "npm run check:dotnet",
-          "dotnet build apps/dh/api-dh/DataHub.WebApi.sln --configuration=Release"
-        ],
-        "parallel": false
-      },
-      "outputs": ["{projectRoot}/source/DataHub.WebApi/bin/Release/net8.0"]
+        "command": "dotnet build apps/dh/api-dh/DataHub.WebApi.sln --configuration=Release"
+      }
     },
     "build-client": {
-      "dependsOn": [
-        "build"
-      ],
+      "dependsOn": ["build"],
+      "outputs": ["{workspaceRoot}/libs/dh/shared/domain/src/lib/generated/v1"],
       "executor": "nx:run-commands",
       "options": {
         "commands": [
-          "yarn check:java",
           "yarn openapi-generator-cli generate -o libs/dh/shared/domain/src/lib/generated/v1 -i apps/dh/api-dh/source/DataHub.WebApi/bin/Release/net8.0/swagger.json -g typescript-angular -p=fileNaming=kebab-case -p=serviceFileSuffix=-http.service -p=serviceSuffix=Http -p=supportsES6=true -p=taggedUnions=true -p=enumPropertyNaming=original",
```

Comment:
Suggestion: Leave in check:java because openapi-generator-cli has it as a dependency. Or should it just be allowed to fail?

Reply:
I brought it back: https://github.com/Energinet-DataHub/greenforce-frontend/pull/2792/commits/4ef6b9b78781760fc969c744b4e7deab165f3e03